Create a Resource for people to educate themselves

DonRichards opened the pull request:
This is to add value to the conversation but not to add directly to the proposal. This could go out of control so I added parameters in the form of a key

DonRichards committed:
Create a Resource for people to educate themselves

This is to add value to the conversation but not to add directly to the proposal. This could go out of control so I added parameters in the form of a key

jonstokes commented:
I like this a lot.

For categories, I can see how that could be contentious. I suggest the following:
- Reference: this is for straight-up reference type stuff, which some still may flag for containing "opinions" about thing but that nonetheless is aimed at being informational.
- Advocacy: this would be how we'd flag advocacy journalism, fact sheets from sources like the NRA, and stuff that's clearly designed to sway opinion in one direction or another.
- Opinion: the standard for this category would be stuff that's clearly marked as "opinion" by the publisher, i.e. op-eds and editorials in major newspapers.
- Discussion: this would be for forum threads, reddit threads, HN discussions, and so on that are worth looking at.

I think my original piece on the AR-15 would go under "advocacy" in the above, because though it includes a lot of info, the point is to make a case for why someone without military pretensions would choose the AR-15 over some other weapon type on practical grounds.

DonRichards committed:
Update Keys

Add Reference, & Discussion
Add additional stipulations for opinionated to include "opinion"  marked by publisher.

DonRichards commented:
I think I mixed the suggestions into this without adding too many new categories. Your thoughts?

jonstokes commented:
Actually after I merged I realized that this was a PR to a branch that's already merged into master. I'll fix it.
